package.json

```diff
@@ -33,6 +34,12 @@
     "yosay": "^1.2.1"
   },
   "devDependencies": {
-    "mocha": "*"
+    "mocha": "*",
+    "eslint": "^4.11.0",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-loader": "^1.9.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jsx-a11y": "^6.0.2",
```

Comment:
Also, what are we doing with accessibility in here?

Reply:
It's a requirement for the airbnb linter. A number of rules that airbnb uses exist already in jsx-a11y, so instead of duplicating them, they pull in that package.